src/SmartHotel.Clients/SmartHotel.Clients/Services/IoT/IRoomDevicesDataService.cs

```diff
@@ -6,8 +6,8 @@ public interface IRoomDevicesDataService
 	{
 		bool UseFakes { get; }
 
-		Task<RoomAmbientLight> GetRoomAmbientLightAsync();
-		Task<RoomTemperature> GetRoomTemperatureAsync();
+		Task<RoomAmbientLight> GetRoomAmbientLightAsync(string token = "");
+		Task<RoomTemperature> GetRoomTemperatureAsync(string token = "");
```

Comment:
These 2 methods should not be taking in a token. I'll explain in a different comment